src/GraphQL/Resolvers/Resolver.php

```diff
@@ -59,7 +59,8 @@ public static function resolveAddElementToArea(
     ): BaseElement {
         $elementClass = $args['className'];
         $elementalAreaID = $args['elementalAreaID'];
-        $afterElementID = $args['afterElementID'] ?? null;
+        // Remove areaaID suffix from 0_${areaaID} value and leave as it is ${elementID} value. We use it in line 99
+        $afterElementID = explode('_',$args['afterElementID'])[0] ?? null;
```

Comment:
I really don't like this. We are changing the data that is being sent to the server in order to fix a UI bug.

Reply:
A small note also about the comment - it's best to avoid mentioning line numbers in a comment, because as soon as someone adds or removes a line of code that comment will no longer be correct.

Reply:
DONE